### PR TITLE
move isort configuration to pyproject

### LINKED ISSUE #27: Fourmat overrides .isort.cfg when project specifies [tool.isort] configuration in pyproject.toml
Modern versions of `isort` allow developers to specify configuration using a `[tool.isort]` section in `pyproject.toml`. This is preferable to prevent an over-proliferation of config file types, but `fourmat` will only see that an `.isort.cfg` file is missing and supplant its own default configuration, which takes precedence over the `pyproject.toml` configuration.

### PR DESCRIPTION
**Pull request by itajaja**
closes #27

cc @ccarpita

### CODE REVIEW

File: fourmat/lint.py [hunk 1]
```diff
@@ -25,32 +24,6 @@ def get_project_paths():
     return CONFIG_FILE.read_text().split()
 
 
-def get_dirty_filenames(paths, *, staged=False):
```
Comment (itajaja) [inline fourmat/lint.py:28]: DCE

**Review by c0state — APPROVED** [review]
lgtm

File: fourmat/lint.py (hunk 1 above)
Comment (c0state) [reply fourmat/lint.py:28]: > DCE

?

Comment (itajaja) [reply fourmat/lint.py:28]: dead code elimination 😅

**Pull request merged**